mixins/test/test-elem.js

```diff
@@ -5,7 +5,7 @@ class TestElem extends LocalizeMixin(LitElement) {
 
 	static get properties() {
 		return {
-			date: { type: Date },
+			date: { attribute: false },
```

Comment:
`Date` isn't a valid property type. But if we don't bind this to an attribute, it's OK.